packages/nextjs/src/adapter-nextjs/getNewsroomServerSideProps.ts

```diff
@@ -20,12 +21,15 @@ export async function getNewsroomServerSideProps(
     assertServerEnv('getNewsroomServerSideProps');
     const { req: request, locale } = context;
 
-    const api = getNextPrezlyApi(request);
-    const serverSideProps = await api.getNewsroomServerSideProps(request, locale, options.story);
+    const api = NextContentDelivery.initClient(request, {
+        pinning: options.pinning,
+    });
 
-    if (options.loadHomepageContacts) {
-        serverSideProps.newsroomContextProps.contacts = await api.getNewsroomContacts();
-    }
```

Comment:
I wonder why it was done this way. By moving this call into `api.getNewsroomServerSideProps()` it can be fetched concurrently, and be faster.